clientlibrary/service/discovery_android.go

```diff
@@ -76,8 +77,15 @@ func (d *discoveryObserver) ServiceType() string {
 }
 
 func (d *discoveryObserver) ObserveChange(result ObservationResult) {
+	// in the newer android API it can return multiple IPs separated by comma
+	// sorry, slices are not supported in the bridge :'(
+	var ips = strings.Split(result.Ip()[0], ",")
```

Comment:
code style: `ips := `

Reply:
fixed. Also it was a copilot introduced bug 😅